sysboundary/sysboundary.cpp

```diff
@@ -523,6 +525,64 @@ bool SysBoundary::classifyCells(dccrg::Dccrg<spatial_cell::SpatialCell,dccrg::Ca
    
    technicalGrid.updateGhostCells();
    
+   const std::array<int,3> fsGridDimensions = {convert<int>(P::xcells_ini) * pow(2,P::amrMaxSpatialRefLevel),
+      convert<int>(P::ycells_ini) * pow(2,P::amrMaxSpatialRefLevel),
+      convert<int>(P::zcells_ini) * pow(2,P::amrMaxSpatialRefLevel)};
```

Comment:
Perhaps use fsgrid globalsize here instead?

Reply:
I took what was used to initialise the fsgrids, but indeed, this is a better suggestion.

Reply:
But there is no getGlobalSize in fsgrid...

Reply:
That's been implemented.